src/main/java/com/puppycrawl/tools/checkstyle/xpath/XpathQueryGenerator.java

```diff
@@ -332,4 +332,62 @@ private boolean isMatchingByLineAndColumnAndTokenType(DetailAST ast) {
                 && expandedTabColumn(ast) == columnNumber
                 && (tokenType == 0 || tokenType == ast.getType());
     }
+
+    /**
+     * Controls true escape characters.
+     *
+     * @param chr character from parent method encode() .
+     * @param string StringBuilder from parent method encode().
+     */
+    private static void generateControlledString(char chr, StringBuilder string) {
```

Comment:
just `encode` can be ok I guess

Reply:
`encode` is already the main function

Reply:
yeah, but signature is different so it will be overloaded method

Reply:
Yup, got that. Sir, I had changed it to `controlTrueCharacter()` to give a better sense as `encode` might confuse later if some other contributors try to fix something. Will it be ok?